### PR TITLE
Digital Clock

### PR DESCRIPTION
**Pull request by nik132-eng**
# Pull Request for Play With Python 💡

## Issue Title : Digital Clock #94 

- **Info about the related issue (Aim of the project)** : Show time in digital clock
- **Name:** Nikunj Rohit
- **GitHub ID:** nik132-eng
- **Email ID:** 19itnik132@ldce.ac.in
- **Idenitfy yourself: (Mention in which program you are contributing in.)** 

Closes: #94 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by nik132-eng** [comment]
can anyone please check this PR?

**Comment by joleyasarthak** [comment]
Hey @nik132-eng, as a past contributor of this repo, I suggest you to please make a folder, and add your work there, in a proper structure. Please refer the [This](https://github.com/World-of-ML/Play-With-Python#structure-of-the-projects-), as to how you should structure. Hope it helps.

**Comment by nik132-eng** [comment]
ok thank you I will doing this

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Please follow the project structure and add files accordingly.
@nik132-eng

**Comment by abhisheks008** [comment]
Project structure will be like this,
```
Digital Clock
|-- Images
    |-- image1.png
    |-- image2.png
|-- main.py
|-- README.md
|-- requirements.txt
```

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Add the **Images** folder.
2. Rename the `readme_template.md` file as `README.md`.
3. Add the demonstration images inside the README.md file under the Demonstration heading. To add the images in the README.md file, you can use this syntax, ``.

**Comment by nik132-eng** [comment]
Now this is Ok?

File: Digital clock/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**WHAT I HAD DONE**
+
+Write a python code fetch data from computer hardware and show on screen.
+
+**LIBRARIES NEEDED**
+
+ tkinter
+
+**DEMONSTRATION**
+
```
Comment (abhisheks008) [inline Digital clock/readme.md:20]: Please add the demonstration images here,
syntax for adding images, `image name`

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Review by abhisheks008 — APPROVED** [review]
LGTM! Your PR is approved and ready to be merged.
@nik132-eng

**Pull request merged**